pyproject.toml

```diff
@@ -30,3 +30,29 @@ mypy = "^1.10.0"
 [build-system]
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.isort]
+src_paths = ["UTMB-IIS"]
+known_local_folder = ["UTMB-IIS/"]
+line_length = 120
+skip = ["wfastcgi.py"]
```

Comment:
Why this file is exclude in most of linters?

Reply:
its package for deploy on IIS